### ISSUE TITLE
Function: Train Test split 

### ISSUE THREAD

**Issue by rogerkuou** [VegeWaterDynamics/motrainer#93]
Implementation in `motrainer.py`
- [ ] If specify {"coords":mask} or {"dims":mask}, split according to mask 
- [ ] Else, wrap kwargs to `dask_ml.model_selection.train_test_split`
